pychromecast/discovery.py

```diff
@@ -84,8 +85,13 @@ def start_discovery(callback=None):
     ServiceBrowser object.
     """
     listener = CastListener(callback)
-    return listener, \
-        ServiceBrowser(Zeroconf(), "_googlecast._tcp.local.", listener)
+    sb = False
+    try:
+        sb = ServiceBrowser(Zeroconf(), "_googlecast._tcp.local.", listener)
+    except Exception:
+        return listener, False
+    finally:
+        return listener, sb
```

Comment:
⚠️because of this statement in `finally`, `return listener, False` will never be returned.

![carbon 3](https://user-images.githubusercontent.com/6833237/36220721-f7d52a56-11bb-11e8-861c-9ac5a70a52ae.png)

I mean it doesn't really affect the return value, it's still misleading though.